config/locales/ja.yml

```
@@ -15,22 +15,16 @@ ja:
         restrict_dependent_destroy:
           has_one: "%{record}が存在しているので削除できません"
           has_many: "%{record}が存在しているので削除できません"
-  books:
-    index:
-      Books: '本の一覧'
-      Show_this_book: 'この本を表示'
-      New_book: '新規作成'
-    new:
-      New_book: '新規作成'
-      Back_to_books: '本の一覧に戻る'
-    show:
-      Edit_this_book: 'この本を編集'
-      Back_to_books: '本の一覧に戻る'
-      Destroy_this_book: 'この本を消去'
-    edit:
-      Editing_book: '本の編集'
-      Show_this_book: 'この本を表示'
-      Back_to_books: '本の一覧に戻る'
+  views:
+    common:
+      show: 'この%{name}を表示'
+      new: '新しく%{name}を作成'
+      edit: 'この%{name}を編集'
+      back: '%{name}の一覧に戻る'
+      destroy: 'この%{name}を消去'
+      title_index: '%{name}の一覧'
```

```suggestion
      title:
        index: '%{name}の一覧'
```

`title_`と毎回付けるぐらいなら階層化した方がシンプルかも？